code/modules/mob/living/carbon/human/species/_species.dm

```diff
@@ -176,7 +176,17 @@
 	var/datum/language/species_language = GLOB.all_languages[language]
 	return species_language.get_random_name(gender)
 
-/datum/species/proc/create_organs(mob/living/carbon/human/H) //Handles creation of mob organs.
+/datum/species/proc/create_organs(mob/living/carbon/human/H, special = FALSE) //Handles creation of mob organs.
+	var/list/special_sauce = list()
+	if(special)
+		for(var/obj/item/organ/internal/I in H.internal_organs)
+			if(I.special)
+				if(I.status == ORGAN_DEAD|ORGAN_ROBOT) // cybernetic + broken
```

Comment:
Talked about this on discord, but just in case it gets lost.. these are bitflags, so gotta write it like this.
```suggestion
				if(I.status & (ORGAN_DEAD | ORGAN_ROBOT)) // cybernetic + broken
```